example/service/ec2/instancesbyRegion/README.md

```diff
@@ -0,0 +1,28 @@
+
+# Example  Fetch By region
+
+This is an example using the AWS SDK for Go to list ec2 instances instance state By different region . By default it fetch all running and stopped instance 
+
+
+# Usage
+
+
+```sh
+go run  filter_ec2_by_region running
+```
+
+Output:
+```
+
+Fetching instace details  for region:  ap-south-1
+printing instance details.....
+instance id i-00cf3fcsssdd373766
```

Comment:
@xibz , what mock value you mean to say? Are you talking about any unit test with some mock value?